apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.mm

```diff
@@ -0,0 +1,108 @@
+#import "AppDelegate.h"
```

Comment:
We deleted AppDelegate.m file and change to mm file?   Where are we registering extensions for ios  now?